components/Map/Map.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+
+import MapView, { Marker } from 'react-native-maps';
+import { StyleSheet, Text, View, Button, TextInput } from 'react-native';
+
+export default function Map() {
+
+let events = [
+    {
+      id: 1,
+      date: 20200407,
+      time: 1700,
+      unix_time: 1586278800,
```

Comment:
Not sure how the unix time will tie into the BE of our project. I want to discuss this further as a team.

Reply:
Yeah I think they said for sorting future events and placing those in order, but we can double check with them